.eslintignore

```diff
@@ -4,3 +4,4 @@
 /tests/backstop/html_report/*
 /php-scoper/*
 /vendor/*
+/gulpfile.js/*
```

Comment:
This file shouldn't be excluded from linting. Let's either fix the violations, or alternatively we could also disable any JS doc-related rules from applying to any of the gulp files (not much point in documenting parameters etc as thoroughly as we do in production code).

Reply:
Eslint issues have been fixed.